cps_score/cps_score.py

```diff
@@ -787,9 +782,52 @@ def _sponsor_vote(self, _ipfs_key: str, _vote: str, _vote_reason: str, _from: Ad
         else:
             self._remove_contributor(_contributor_address)
             self._update_proposal_status(_ipfs_key, self._REJECTED)
+            # Return 50% of fees back to contributor
+            self.icx.transfer(_contributor_address, to_loop(APPLICATION_FEE) // 2)
+
             self.SponsorBondRejected(_from,
                                      f"Sponsor Bond Rejected for project {_proposal_details[PROJECT_TITLE]}.")
 
+    @external(readonly=True)
+    def getBudgetAdjustmentFeature(self) -> bool:
+        return self.budgetAdjustment.get()
+
+    @external
+    def toggleBudgetAdjustmentFeature(self):
+        self.budgetAdjustment.set(not self.budgetAdjustment.get())
```

Comment:
```suggestion
        self._validate_admins()
        self.budgetAdjustment.set(not self.budgetAdjustment.get())
```

I don't think that you want anyone to toggle budgetAdjustment flag.

Reply:
Right but CPS Score is already live, and this can be holdback until java update may be. ?